merkle/src/merkle.rs

```diff
@@ -20,6 +21,13 @@ use tempfile::tempfile;
 /// as less as possible with multiple threads competing to get the write lock.
 pub const SMALL_TREE_BUILD: usize = 1024;
 
+// Number of batched nodes processed and stored together in `populate_leaves` to
+// avoid single `push`es which degrades performance for `DiskStore`.
+pub const BUILD_LEAVES_BLOCK_SIZE: usize = 1024;
+
+// Number of nodes to process in parallel during the `build` stage.
+pub const BUILD_CHUNK_NODES: usize = 64 * 1024;
```

Comment:
These two constants might as well be collapsed into one (with the appropriate byte-to-node conversion).

Reply:
could you not just do `BUILD_CHUNK_NODES = 64 * BUILD_LEAVES_BLOCK_SIZE`?